ee/tuf/library_lookup.go

```diff
@@ -64,6 +65,21 @@ func ShouldUseNewAutoupdater(ctx context.Context) bool {
 // getAutoupdateConfig pulls the configuration values necessary to work with the autoupdate library
 // from either the given args or from the config file.
 func getAutoupdateConfig(args []string) (*autoupdateConfig, error) {
+	// pflag, while mostly great for our usecase here, expects getopt-style flags, which means
+	// it doesn't support the Golang standard of using single and double dashes interchangeably
+	// for flags. (e.g., pflag cannot parse `-config`, but Golang treats `-config` the same as
+	// `--config`.) This transforms all single-dash args to double-dashes so that pflag can parse
+	// them as expected.
+	argsToParse := make([]string, len(args))
```

Comment:
seems fine (well, as much as any hack) but why not modify `args` in place?

Reply:
It felt a little more cautious to not modify `os.Args`

Reply:
Is it os.Args? I guess I'm not sure how the references/values work for the `args []string` in the signature

Reply:
It is `os.Args`, and modifying `args` here would modify `os.Args`

Reply:
(https://play.golang.com/p/IcOt9XjGcvb if you'd like to see)

Reply:
Want me to add a comment in the code re: why I'm not modifying in place?

Reply:
Well, I guess arrays in Go are all references, so there we go. 

I don't know that it needs comment.  🤷